kingfisher_scrapy/spiders/kosovo.py

```diff
@@ -8,37 +10,36 @@ class Kosovo(SimpleSpider):
       Public Procurement Regulatory Commission
     Spider arguments
       from_date
-        Download only data from this time onward (YYYY-MM-DDThh:mm:ss format).
-        If ``until_date`` is provided, defaults to '2000-01-01T00:00:00'.
+        Download only data from this time onward (YYYY-MM-DD format). Defaults to '2000-01-01'.
       until_date
-        Download only data until this time (YYYY-MM-DDThh:mm:ss format).
-        If ``from_date`` is provided, defaults to now.
+        Download only data until this time (YYYY-MM-DD format). Defaults to now.
     API documentation
-      https://ocdskrpp-test.rks-gov.net/Help
+      https://ocdskrpp.rks-gov.net/Help
     """
     name = 'kosovo'
 
     # BaseSpider
-    date_format = 'datetime'
-    default_from_date = '2000-01-01T00:00:00'
-    root_path = 'item'
+    date_format = 'date'
+    date_required = True
+    default_from_date = '2000-01-01'
 
     # SimpleSpider
-    data_type = 'release'
+    data_type = 'release_package'
 
     def start_requests(self):
-        stages = ['Award', 'Tender', 'Bid']
-        url = 'https://ocdskrpp-test.rks-gov.net/krppAPI/{}'
-        headers = {'accept': 'application/json', 'Content-Type': 'application/json'}
-
-        if self.from_date and self.until_date:
-            from_date = self.from_date.strftime(self.date_format)
-            until_date = self.until_date.strftime(self.date_format)
-            url = f'{url}?endDateFrom={from_date}&endDateEnd={until_date}'
-
-        for stage in stages:
-            yield self.build_request(
-                url.format(stage),
-                headers=headers,
-                formatter=components(-1)
-            )
+        # The API is slow even with short periods, so we request one day at a time.
+        delta = self.until_date - self.from_date
+        for days in reversed(range(delta.days + 1)):
+            start = self.from_date + timedelta(days=days-1)
+            end = self.from_date + timedelta(days=days)
+            url = f'https://ocdskrpp.rks-gov.net/krppapi/tenderrelease?endDateFrom=' \
```

Comment:
I would have expected flake8 to complain about an f-string without a placeholder.

```suggestion
            url = 'https://ocdskrpp.rks-gov.net/krppapi/tenderrelease?endDateFrom=' \
```